Run sdks java example

```
Exception in thread "main" java.lang.UnsatisfiedLinkError: could not load a native library: frontend
        at io.netty.util.internal.NativeLibraryLoader.load(NativeLibraryLoader.java:239)
        at com.automq.elasticstream.client.jni.Frontend.loadNativeLibrary(Frontend.java:21)
        at com.automq.elasticstream.client.jni.Frontend.<clinit>(Frontend.java:14)
        at com.automq.elasticstream.client.DefaultStreamClient.<init>(DefaultStreamClient.java:17)
        at com.automq.elasticstream.client.DefaultClient.<init>(DefaultClient.java:13)
        at com.automq.elasticstream.client.DefaultClientBuilder.build(DefaultClientBuilder.java:39)
        at com.automq.elasticstream.client.examples.Main.main(Main.java:18)
Caused by: java.io.FileNotFoundException: META-INF/native/libfrontend.so
        at io.netty.util.internal.NativeLibraryLoader.load(NativeLibraryLoader.java:186)
        ... 6 more
        Suppressed: java.lang.UnsatisfiedLinkError: no frontend in java.library.path
                at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1860)
                at java.lang.Runtime.loadLibrary0(Runtime.java:843)
                at java.lang.System.loadLibrary(System.java:1136)
                at io.netty.util.internal.NativeLibraryUtil.loadLibrary(NativeLibraryUtil.java:38)
                at io.netty.util.internal.NativeLibraryLoader.loadLibrary(NativeLibraryLoader.java:391)
                at io.netty.util.internal.NativeLibraryLoader.load(NativeLibraryLoader.java:161)
                ... 6 more
                Suppressed: java.lang.UnsatisfiedLinkError: no frontend in java.library.path
```

![image](https://github.com/AutoMQ/elastic-stream/assets/5719209/d24a14bc-ac13-4f93-b575-a2c7d27730da)

--- a/sdks/frontend-rs/Cargo.toml
+++ b/sdks/frontend-rs/Cargo.toml
@@ -30,4 +30,4 @@ test-util = { path = "../../components/test-util" }
 minitrace = { workspace = true }
 
 [lib]
-crate-type = ["lib"]
+crate-type = ["dylib"]

--- a/sdks/frontend-rs/Cargo.toml
+++ b/sdks/frontend-rs/Cargo.toml
@@ -30,4 +30,4 @@ test-util = { path = "../../components/test-util" }
 minitrace = { workspace = true }
 
 [lib]
-crate-type = ["cdylib", "rlib"]
+crate-type = ["cdylib", "lib"]

--- a/sdks/frontend-rs/Cargo.toml
+++ b/sdks/frontend-rs/Cargo.toml
@@ -30,4 +30,4 @@ test-util = { path = "../../components/test-util" }
 minitrace = { workspace = true }
 
 [lib]
-crate-type = ["dylib"]
+crate-type = ["cdylib", "rlib"]

